lib/dialects/sqlite/query-interface.js

```diff
@@ -130,50 +136,53 @@ function removeConstraint(tableName, constraintName, options) {
       });
     })
     .then(fields => {
-      const sql = this.QueryGenerator._alterConstraintQuery(tableName, fields, createTableSql);
+      const sql = qi.QueryGenerator._alterConstraintQuery(tableName, fields, createTableSql);
       const subQueries = sql.split(';').filter(q => q !== '');
 
-      return Promise.each(subQueries, subQuery => this.sequelize.query(`${subQuery};`, Object.assign({ raw: true }, options)));
+      return Promise.each(subQueries, subQuery => qi.sequelize.query(`${subQuery};`, Object.assign({ raw: true }, options)));
     });
 }
 exports.removeConstraint = removeConstraint;
 
-function addConstraint(tableName, options) {
-  const constraintSnippet = this.QueryGenerator.getConstraintSnippet(tableName, options);
-  const describeCreateTableSql = this.QueryGenerator.describeCreateTableQuery(tableName);
+/**
+ * @param {QueryInterface} qi
+ */
```

Comment:
```suggestion
 * @private
 */
```

Reply:
For other methods as well, or just remove the new jsdoc headers from these dialect based query interface methods